ethers-solc/src/lib.rs

```diff
@@ -426,6 +426,36 @@ impl<T: ArtifactOutput> Project<T> {
     pub fn flatten(&self, target: &Path) -> Result<String> {
         self.paths.flatten(target)
     }
+
+    /// Returns standard-json-input to compile the target contract
+    pub fn standard_json_input(&self, target: &Path) -> Result<String> {
+        tracing::trace!("Building standard-json-input");
+        let graph = Graph::resolve(&self.paths)?;
+        let target_index = graph.files().get(target).ok_or_else(|| {
+            SolcError::msg(format!("cannot resolve file at \"{:?}\"", target.display()))
+        })?;
+        let mut sources = Vec::new();
+        let (path, source) = graph.node(*target_index).unpack();
```

Comment:
should probably work
```suggestion
        let (path, source) = graph.node(*target_index).unpack().clone();
```